src/modules/routes/constants/views.js

```diff
@@ -10,6 +10,7 @@ export const CONNECT = 'connect'
 export const CREATE = 'create'
 export const CATEGORIES = 'categories'
 export const REPORTING = 'reporting'
+export const REPORT = 'report'
```

Comment:
You've been renaming `REPORTING_REPORT` just to `REPORT` generally, right? So here, instead of adding this line, I think you actually want to edit line 46 (`export const REPORTING_REPORT = 'reporting-report'`).